pwd-service/src/main/resources/application.yml

```diff
@@ -14,6 +14,12 @@ spring:
 analysis:
   endpoint: http://pwd-analysis.herokuapp.com/analysis
 
+smtp:
+  mailbox: kontakt@pwd.dolinagubra.pl
+
+MAILGUN_API_KEY: dummyApiKey
```

Comment:
niestety, nie udało mi się założyć nowych propertiesów - może to kwestia uprawnień?